sunpy/net/fido_factory.py

```diff
@@ -492,7 +492,7 @@ def _make_query_to_client(self, *query):
         return results
 
     def __repr__(self):
-        return object.__repr__(self) + "\n" + self._print_clients(visible_entries=15)
```

Comment:
The fact this was explicitly set makes it feel like a choice. I am going to ping @nabobalis for his opinion as I am assuming he wrote this.

Reply:
I wanted to prevent too much table being printed but if you want to print it all, print it all.